src/ui/ResultLayout/ResultLayout.ts

```diff
@@ -37,6 +39,7 @@ export class ResultLayout extends Component {
    * The possible valid and supported layout.
    */
   public static validLayouts: ValidLayout[] = ['list', 'card', 'table'];
+
   /**
    * The current active layout.
```

Comment:
Hum, the getCurrentLayout() is a bit redundant with the attribute. We could remove it from the doc, yeah.